spotbugs/src/main/java/edu/umd/cs/findbugs/BugInstance.java

```diff
@@ -100,6 +104,8 @@
  */
 public class BugInstance implements Comparable<BugInstance>, XMLWriteable, Cloneable {
 
+    private static final Logger LOG = LoggerFactory.getLogger(BugInstance.class);
```

Comment:
This could be made copy-paste compatible using `MethodHandles`, eg:

    private static final Logger LOG = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());